src/niveristand/library/primitives.py

```diff
@@ -215,7 +215,7 @@ def tickcountms():
         int: time, in milliseconds, as reported by the high-precision counter (if available).
 
     """
-    return int(time.clock() * 10 ** 3)
+    return int(time.process_time() * 10 ** 3)
```

Comment:
You should use the `time.perf_counter()` to actually get the value of the counter.